src/content/docs/en/guides/markdown-content.mdx

```diff
@@ -603,6 +603,10 @@ export default defineConfig({
       langs: [],
       // Enable word wrap to prevent horizontal scrolling
       wrap: true,
+      // Add custom tranformers
+      // Find common ones from https://shikiji.netlify.app/packages/transformers
+      // https://shikiji.netlify.app/guide/transformers
```

Comment:
```suggestion
      // Add custom transformers: https://shikiji.netlify.app/guide/transformers
      // Find common transformers: https://shikiji.netlify.app/packages/transformers
```
I think we can make this a little more compact, and include the link to the guide with the mention of creating custom ones!